ui/widgets/nuxeo-actions-menu.js

```diff
@@ -222,7 +222,7 @@ import './nuxeo-tooltip.js';
          * updated on polyfilled browsers (Firex and Edge), leading to an empty menu if there's a single element
          * that doesn't fit on the menu.
          */
-        while (els.length && this.contentWidth <= this.clientWidth) {
+        while (els.length && this.contentWidth < this.clientWidth) {
```

Comment:
What is the reasoning behind this change?

Reply:
There is an issue with the navigation bar when it extended the size then it makes scrollable. So to resolve this issue we need to change this condition.

https://jira.nuxeo.com/secure/attachment/118464/bugActionMenu.webm